client/stylesheets/_toCat.scss

```diff
@@ -5451,7 +5466,7 @@ On click the images will be displayed at normal size to complete the effect
 /** Amount Profile **/
 
 .profileUserAmount ul {
-    margin: 8px 0 !important;
+    margin: 15px 0 8px 0 !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces<br>Shorthand form for property `margin` should be written more concisely as `15px 0 8px` instead of `15px 0 8px 0`